projects/ansible/collections/dev_guidelines/releasing_collections_without_release_branches.rst

```diff
@@ -0,0 +1,93 @@
+**********************************************
+Releasing collections without Release Branches
+**********************************************
+
+If you have any question when reading this manual, ask the community in the ``#ansible-community`` IRC channel or create an issue in the `community-docs <https://github.com/ansible/community-docs>`_ repository.
+
+This manual assumes that publishing the collection is done via `Zuul <https://github.com/ansible/project-config>`_ and that `antsibull-changelog <https://github.com/ansible-community/antsibull-changelog>`_ is used for the changelog.
+
+Since no release branches are used, the manual does not distinguish between releasing a major, minor, or patch version.
+
+1. First, examine the collection if there are merged changes to release.
+
+2. According to the changes made, choose an appropriate release version number. Keep in mind that the collections must follow the `semantic versioning <https://semver.org/>`_ rules. Refer to the `Releasing without release branches <releasing_collections_without_release_branches.rst>`_ for details.
+
+3. Announce your intention to release the collection in a corresponding pinned release issue / community pinboard of the collection and in the ``#ansible-community`` IRC channel.
+
+4. Be sure you are in a default branch in your local fork:
+
+.. code:: bash
+
+    git status
+    git checkout DEFAULT_BRANCH     # if needed
+
+5. Update your local fork:
+
+.. code:: bash
+
+   git pull --rebase upstream DEFAULT_BRANCH
+
+6. Checkout a new release branch from the default branch:
+
+.. code:: bash
+
+   git checkout -b release_branch
+
+7. Make sure that ``galaxy.yml`` contains the correct release version number. If not, update it!
+
+8. Add a changelog fragment ``changelogs/fragments/X.Y.Z.yml`` with content:
+
+.. code:: yaml
+
+   release_summary: |-
+     Write some text here that should appear as the release summary for this version.
+     The format is reStructuredText (but not a list as for regular changelog fragments).
+     This text will be verbatimly inserted into the changelog.
+
+For example:
+
+.. code:: yaml
+
+   release_summary: |-
+     This is the minor release of the ``community.mysql`` collection.
+     This changelog contains all changes to the modules and plugins in this collection
+     that have been made after the release of ``community.mysql`` 1.2.0.
```

Comment:
I would use something like:
```suggestion
     that have been made after the previous release.
```
to avoid having to update the version in this text manually. Makes copy'n'paste errors less frequently :)

Reply:
sounds sensible